public/js/performances/new.js

```diff
@@ -16,10 +18,10 @@ $('.NewPerformanceSubmit').on('click', () => {
       credentials: 'same-origin',
       method: 'post',
       body: JSON.stringify({
-        closeAt: new Date(inputs.closeAt.val()).toISOString(),
+        closeAt: moment.utc(new Date(inputs.closeAt.val())).format(),
```

Comment:
Why the switch to `moment.utc`? The only difference should be the millisecond precision of `toISOString()`, which shouldn't be an issue, should it?